datameta/api/files.py

```diff
@@ -36,12 +38,14 @@ class FileBase(DataHolderBase):
     user_id     : str
```

Comment:
hah, this is not properly formatted :D

Reply:
It is with the ignore list specified in .flake8 😇